docs/displaysystem/angular.md

```diff
@@ -20,32 +20,28 @@ limitations under the License.
 {% include JB/setup %}
 
 
-### Angular (Beta)
+## Angular Display System in Zeppelin
 
 Angular display system treats output as an view template of [AngularJS](https://angularjs.org/).
-It compiles templates and display inside of Zeppelin.
+It compiles templates and displays inside of Zeppelin.
```

Comment:
It compiles templates and displays inside of Zeppelin
-> It compiles templates and displays them inside of Zeppelin